lib/elixir/lib/option_parser.ex

```diff
@@ -479,7 +479,7 @@ defmodule OptionParser do
       ["--bool", "--no-bool"]
 
   Some switches will output different values based on the switches
```

Comment:
“based on the `:switches` option”?  In this case, `:switches` is one of the options of `to_argv`.

Reply:
I think it is ok as it is, since switches type is a subset of `:switches` option

Reply:
in this case if it refering to `:count`  in particular, which is referred as a "type"